File: lab_operadores/lab_operadores/csharp/Decision.cs
using System;
class Decision {
    static void Main() {
        int nota = 72;
        if (nota >= 70) Console.WriteLine($"Aprovado com {nota}");
        else if (nota >= 40) Console.WriteLine($"Recuperacao com {nota}");
        else Console.WriteLine($"Reprovado com {nota}");
    }
}

File: lab_operadores/lab_operadores/c/decision.c
#include <stdio.h>
int main() {
    int nota = 72;
    if (nota >= 70) printf("Aprovado com %d\n", nota);
    else if (nota >= 40) printf("Recuperacao com %d\n", nota);
    else printf("Reprovado com %d\n", nota);
    return 0;
}